t04_refactored.py

```diff
@@ -170,8 +174,52 @@ def team_12_adv():
 
 
 def team_13_adv():
-    pass
-    # TODO Add your code here
+    """
+    Andy Hill
+    Nyan Lin Zaw
+    https://docs.google.com/document/d/1-lO7SPynvH8GxfrMS5xBYxhiu9YZs-9oWcI3YCdIKTA/edit
+    :return: none
+    """
+
+    from time import sleep
+
+    slow = 1.5
+    pieface = False
+    username = input("What shall we call you? ")  # asking for user's name
+    print("\n\n")
+    print("Welcome," + username + ", to our yearly raffle!")
+    sleep(slow)
+    print("In front of you, we have presented you with five tickets. Choose a number from 1 to 5.")
+    print("Choose carefully!")
+    sleep(slow)
+
+    ticket = input("Which ticket shall you choose? [1/2/3/4/5] ")
+
+    if ticket == "3":
+        # Good Choice
+        print("Congrats! You picked the golden ticket!")
+        print("You have won a delicious chocolate pie! Enjoy!")
+        sleep(slow)
+        pieface = False
+    elif ticket == "5":
+        # Bad Choice
+        print("Oh, no! Someone is out of luck today. Watch out for the flying whipped cream pie!")
+        pieface = True
+        if pieface:
+            print(
+                "This year was not your year. Come back next time. Try again by hitting the green play button.")
+    else:
+        # Neutral Choice
+        print("Oh, no! You picked one of the right tickets, but we ran out of pies :( ")
+        print("But lucky you, we still have some cupcakes! Enjoy!")
+        sleep(slow)
+    quit()
```

Comment:
Don't quit() because it ends the code here always, and there might be other team's adventures after yours. Instead, use the `kill_if_dead()` function here.